internal/bminventory/inventory.go

```diff
@@ -2102,6 +2102,9 @@ func (b *bareMetalInventory) updateClusterData(_ context.Context, cluster *commo
 			&map[string]interface{}{"hyperthreading_enabled": *params.ClusterUpdateParams.Hyperthreading}, usages)
 	}
 
+	b.setUsage(swag.BoolValue(params.ClusterUpdateParams.UserManagedNetworking) &&
```

Comment:
what happens if the user doesn't provide any argument (which should be reflected with ``nil`` in ``params.ClusterUpdateParams.UserManagedNetworking`` but did set ``UserManagedNetworking`` previously in the cluster?
Won't it override it with false?